virttest/utils_misc.py

```diff
@@ -3102,9 +3103,8 @@ def get_image_info(image_file):
     """
     try:
         cmd = "qemu-img info %s" % image_file
-        if compare_qemu_version(2, 10, 0):
-            # Currently the qemu lock is only introduced in qemu-kvm-rhev,
-            # if it's introduced in qemu-kvm, will need to update it here.
+        if libvirt_storage.check_qemu_image_lock_support():
```

Comment:
image lock is not libvirt special feature, condication here is not suitable.

Reply:
@xutian Where do you think it should be?utils_misc?